types/user.ts

```diff
@@ -1,7 +1,10 @@
 import { User as UserModel } from 'server/models';
 import { SerializedModel } from './serializedModel';
 
-type UserType = SerializedModel<UserModel>;
+type UserType = SerializedModel<UserModel> & {
+	// this is useful to prevent repeated checks for admin status
+	isAdminOfThisCommunity?: boolean;
+};
```

Comment:
we need to do admin checks for auth tokens (bc what if someone gets removed as admin for a community)

however, a lot of routes also do these kinds of auth checks, which is redundant. this acts as a sort of "cache" for those checks.

this works if api requests never cross community boundaries, which i'm pretty sure they don't